src/experimental/safe_html_transform/browser/configure.zcml

```diff
@@ -17,4 +17,14 @@
         directory="static"
         />
 
+    <adapter factory=".controlpanel.FilterControlPanelAdapter" />
+
+    <!-- Filter Control Panel -->
+    <browser:page
+      name="controlpanel"
```

Comment:
@prakharjoshi You registered the filter control panel as "controlpanel" and the configlet with "safe_html_transform-settings". I do not think that works at all. Have you tested this code? We should either override the filter-controlpanel view or add an additional view.